recipes/aimrecords/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "aimrecords" %}
+{% set version = "0.0.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/aimhubio/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 3d1559200d699390f9021569919c0a50a63b38167393377756b3908a83648e4e
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - base58
+    - pytest
+    - ipdb
```

Comment:
These are likely only run dependencies

Reply:
tested this out and base58 is required, but the others aren't